packages/spawner/src/escrow.js

```diff
@@ -1,6 +1,6 @@
 // Copyright (C) 2019 Agoric, under Apache License 2.0
 
-/* global harden E makePromiseKit */
+/* global E makePromiseKit */
```

Comment:
Who makes `makePromiseKit` a global? Why?

This need not be answered in this PR.

Reply:
cc @warner or @michaelfig

Reply:
> Who makes `makePromiseKit` a global? Why?

`spawner` has been mostly unmaintained since its `contractHost` days, which preceded Zoe.  So, it has some peculiarities like this, where it is endowing several "useful globals" to the spawned code.